llvm/include/llvm/TargetParser/RISCVTargetParser.h

```diff
@@ -69,9 +69,16 @@ inline static bool isValidLMUL(unsigned LMUL, bool Fractional) {
   return isPowerOf2_32(LMUL) && LMUL <= 8 && (!Fractional || LMUL != 1);
 }
 
+// Is this a EDIV value that can be encoded into the VTYPE format.
+inline static bool isValidEDIV(unsigned EDIV) {
```

Comment:
IIRC, C906/C910 didn't implement EDIV.

Reply:
Yes, I also think it should not be implemented because there is no hardware to support it.